### PR TITLE
docs: Update introduction topic

### PR DESCRIPTION
**Pull request by aikithoughts**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Current introduction topic for the documentation doesn't contain a lot of information. 

Issue Number: N/A

## What is the new behavior?
We're updating it to provide better guidance for new developers.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/marketing/docs.md [hunk 1]
```diff
@@ -1,3 +1,85 @@
...
+
+* [HTML](https://developer.mozilla.org/docs/Learn/HTML/Introduction_to_HTML "Learn HTML")
+* [CSS](https://developer.mozilla.org/docs/Learn/CSS/First_steps "Learn CSS")
+* [JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript "Learn JavaScript"), including:
+  * [The latest standards](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Language_Resources "Latest JavaScript standards")
+  * [classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes "ES2015 Classes")
+  * [modules](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import "ES2015 Modules")
+* [TypeScript](https://www.typescriptlang.org/ "TypeScript")
+
+## Learn Angular by exploring
```
Comment (kapunahelewong) [inline aio/content/marketing/docs.md:25]: Love these sections!

File: aio/content/marketing/docs.md [hunk 2]
```diff
@@ -1,3 +1,85 @@
...
+1. [Understanding Angular](). While the Try It and Elements of Angular topics provide a good overview of
+the Angular development experience, there many additional features and capabilities of Angular you might want
+to explore. The Understanding Angular section gives you the opportunity to learn more about what you
+can do with Angular's common coding elements.
+
+1. [Developer Guides](). Angular has a number of features and best practices that you can use to build
+all sorts of applications. The Developer Guides section provides you with the information you need to put
+these features and practices to use.
+
+## Learn Angular by reading
```
Comment (kapunahelewong) [inline aio/content/marketing/docs.md:49]: Love this one too! Would the API docs have a place here? Maybe not since it's introductory.

**Review by kapunahelewong — COMMENTED** [review]
Just some preliminary thoughts. This is fun!

**Comment by atscott** [comment]
Closing as obsolete / out of date. Please open a new PR if you plan to make this change in the future.

**Pull request closed without merging**